backend/bin/deal-observer-backend.js

```diff
@@ -23,8 +24,7 @@ if (!INFLUXDB_TOKEN) {
 assert(SPARK_API_BASE_URL, 'SPARK_API_BASE_URL required')
 assert(SPARK_API_TOKEN, 'SPARK_API_TOKEN required')
 
-const OBSERVE_ACTOR_EVENTS_LOOP_INTERVAL = 10 * 1000
-const SPARK_API_SUBMIT_DEALS_LOOP_INTERVAL = 10 * 1000
+const LOOP_INTERVAL = 10 * 1000
```

Comment:
I think we should keep two constants, because we are likely to use different loop delay value for each loop. We already discussed changing `OBSERVE_ACTOR_EVENTS_LOOP_INTERVAL` from 10s to something like 30s or 100s, see https://github.com/CheckerNetwork/spark-deal-observer/pull/29#discussion_r1931754959

Having one constant will make such changes slightly more difficult to implement in the future.

This comment is not blocking this PR from being landed, though.